### PR TITLE
Addition of JUnit RouteMapCommandTest

### LINKED ISSUE #18: Add junit test for RouteMapCommand.java
Edit the junit test to include simulated user input and expected output.

### PR DESCRIPTION
**Pull request by Johnson-Yee**
Fixes #18  
Do note that a Utillity class could be created to initialise a set of bus variables for us to implement in our respective test class.

### CODE REVIEW

**Review by EthanWong2212 — APPROVED** [review]

File: src/test/java/seedu/duke/logic/commands/RouteMapCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        BusStops.UNIVERSITYTOWN,
+        BusStops.UNIVERSITYHEALTHCENTRE,
+        BusStops.OPPKENTRIDGEMRTSTATION,
+        BusStops.PGPR
+    };
+
+    @Test
+    void executeCommand_success() {
+        busesInZoneA.add(new Bus("AA1", AA1));
+        BusData.createBusList(busesInZoneA);
```
Comment (wamikamalik) [inline src/test/java/seedu/duke/logic/commands/RouteMapCommandTest.java:31]: Since you have just one test, using createBusList() here is ok. If you have more than one tests for BusData, remember that calling it again might cause duplicate entries in the list.
A possible way to do such initialization is using a @BeforeAll method.

**Review by wamikamalik — APPROVED** [review]
Looks good to merge!

File: src/test/java/seedu/duke/logic/commands/RouteMapCommandTest.java (hunk 1 above)
Comment (wamikamalik) [reply src/test/java/seedu/duke/logic/commands/RouteMapCommandTest.java:31]: This is something I found from my research. Do let me know if there is a better alternative!

**Comment by wamikamalik** [comment]
Is this supposed to fix the issue #18 ?
